app/src/main/java/com/example/myapplicationwork/MoviesEntityRepository.kt

```diff
@@ -0,0 +1,86 @@
+package com.example.myapplicationwork
+
+import android.content.Context
+import com.example.myapplicationwork.data.Actor
+import com.example.myapplicationwork.data.Genre
+import com.example.myapplicationwork.data.Movie
+import com.example.myapplicationwork.entity.ActorEntity
+import com.example.myapplicationwork.entity.GenreEntity
+import com.example.myapplicationwork.entity.MovieEntity
+
+class MoviesEntityRepository(applicationContext: Context) {
+
+    private val moviesDb = DataBase.create(applicationContext)
```

Comment:
to create dao use Application class 
in this case inside `MoviesEntityRepository` you don't need to context instance